packit_service/worker/build/build_helper.py

```diff
@@ -377,12 +428,27 @@ def _report(
         The status reporting should be done through this method
         so we can extend it in subclasses easily.
         """
-        self.status_reporter.report(
-            description=description,
-            state=state,
-            url=url,
-            check_names=check_names,
-        )
+        if self.is_gitlab_instance and not self.is_reporting_allowed:
```

Comment:
What about switching the order of those two `if` branches to have the shorter one in the beginning?

So, instead of this:
```python
if something:
    # long
else:
    # short
```

use this:

```python
if not something:
    # short
    return

# long
```

Reply:
👍